include/deal.II/base/tensor.h

```diff
@@ -893,14 +893,69 @@ operator Tensor<1,dim,Tensor<rank_-1,dim,OtherNumber> > () const
 }
 
 
+
+namespace internal
+{
+  namespace TensorSubscriptor
+  {
+    namespace
```

Comment:
Why an anonymous namespace? I thought we agreed upon no anonymous namespaces in header files?